source/plugin/data/custom/customdata.rst

```diff
@@ -0,0 +1,296 @@
+===========
+Custom Data
+===========
+
+.. javadoc-import::
+    org.spongepowered.api.data.DataHolder
+    org.spongepowered.api.data.DataManager
+    org.spongepowered.api.data.DataRegistration
+    org.spongepowered.api.data.Key
+    org.spongepowered.api.data.persistence.DataContentUpdater
+    org.spongepowered.api.data.persistence.DataContainer
+    org.spongepowered.api.data.persistence.DataBuilder
+    org.spongepowered.api.data.persistence.DataQuery
+    org.spongepowered.api.data.persistence.DataSerializable
+    org.spongepowered.api.data.persistence.DataStore
+    org.spongepowered.api.data.value.MapValue
+    org.spongepowered.api.data.value.Value
+    org.spongepowered.api.entity.Entity
+    org.spongepowered.api.event.lifecycle.RegisterDataEvent
+
+
+
+The core part of custom data is the :javadoc:`DataSerializable`. To implement it, you must first decide if you want to 
+create a separate API for your custom data. Generally speaking it's best to separate the API from the implementation 
+(as SpongeAPI does), but if it won't be seen by other developers then you can just put both in the same class.
+
+First, create a class and define the data you wish to store. In the following example we will use the idea of storing A
```

Comment:
Why is the A capitalised?

Reply:
Vs:code refused to let me press enter or space without capitalising it. Ill edit in notepad++

Reply:
Updated